### PR TITLE
update yaml for #15334

### PR DESCRIPTION
**Pull request by garfield69**
This updates all the yaml that use the flaresolverr, cookie and useragent info blocks

### CODE REVIEW

**Comment by garfield69** [comment]
dependant on #15334

File: src/Jackett.Common/Definitions/torrentbd.yml [hunk 1]
```diff
@@ -79,16 +79,12 @@ settings:
     type: text
     label: Cookie
   - name: info_cookie
-    type: info
-    label: How to get the Cookie
-    default: "<ol><li>Login to this tracker with your browser</li><li>Ensure you have the <b>Remember me</b> ticked and the <b>Log Me Out if IP Changes</b> unticked when you login</li><li>Open the <b>DevTools</b> panel by pressing <b>F12</b></li><li>Select the <b>Network</b> tab</li><li>Click on the <b>Doc</b> button (Chrome Browser) or <b>HTML</b> button (FireFox)</li><li>Refresh the page by pressing <b>F5</b></li><li>Click on the first row entry</li><li>Select the <b>Headers</b> tab on the Right panel</li><li>Find <b>'cookie:'</b> in the <b>Request Headers</b> section</li><li><b>Select</b> and <b>Copy</b> the whole cookie string <i>(everything after 'cookie: ')</i> and <b>Paste</b> here.</li></ol>"
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/torrentbd.yml:84]: This is different than the rest. Would need to do one of the following:

1) revert to keep the old instructions
2) add a separate info note to tick `Remember me` and not tick `Log Me Out if IP Changes`
3) amend the template in #15335, telling users to tick `Remember me` options, and not tick `Restrict to this IP address` or `Log out after 15 minutes` options

Comment (garfield69) [reply src/Jackett.Common/Definitions/torrentbd.yml:84]: Oh well spotted. Simpler to reverse. We are not bound to use the new method, its simply a convenience not a mandate.

Comment (garfield69) [reply src/Jackett.Common/Definitions/torrentbd.yml]: mind you, that's not a bad idea, to add it to the default for the cookie info, as it should apply to all the other indexer, unless of course there is a site that needs cookie but with remember me force the browser session to logout?

Comment (ilike2burnthing) [reply src/Jackett.Common/Definitions/torrentbd.yml]: I can't think of any like that, but in that rare case we could add a normal info note which tells users not to tick the box.

Comment (garfield69) [reply src/Jackett.Common/Definitions/torrentbd.yml:84]: added the remember-me to cardigann, and set torrentdb back to using enhancement.

**Review by ilike2burnthing — APPROVED** [review]

**Review by ilike2burnthing — APPROVED** [review]

**Pull request merged**